### PR TITLE
Tea 50 create communi tea page

### PR DESCRIPTION
**Pull request by deguzmancory**
### In this PR, I'm adding/changing the following:

Creating the CommuniTEA page.

- [x] Navbar: Add routing to Landing Page and CommuniTEA page.
- [x] Location Search
- [x] Filters Section
- [x] List View Section
- [x] Explore Section
- [x] Footer

### Screenshots / Code Snippets:

Navbar
image

Location Search, FIlters, List View Sections
image

Explore Section
image

Footer
image

### Please do the following to test my code:

1. `git checkout TEA-50-Create-CommuniTEA-page`
2. You _MIGHT_ need to `docker system prune` , `docker compose build`, and `docker compose up` if you face initial errors. 
3. Navigate to `localhost:3000/communitea`

[Related Jira Ticket](https://communiteam.atlassian.net/browse/TEA-50)

### CODE REVIEW

File: .env
Comment (angelajfisher) [inline .env:1]: I think this file got committed before the updated `.gitignore` was pulled from main, so it'll need to be deleted here

File: backend/tmp/main
Comment (angelajfisher) [inline backend/tmp/main:1]: This file should be in the updated `.gitignore` as well, so it should also be deleted here

**Review by angelajfisher — CHANGES_REQUESTED** [review]
LGTM! Just need to delete a couple `.gitignore`-able files before we push to main

File: .env
Comment (deguzmancory) [reply .env:1]: Deleted!

File: backend/tmp/main
Comment (deguzmancory) [reply backend/tmp/main:1]: Deleted!

**Review by angelajfisher — APPROVED** [review]
LGTM! :+1:

**Review by BMLx23 — APPROVED** [review]
Looks good!

**Pull request merged**